app/src/test/java/com/fstyle/structure_android/data/repository/UserRepositoryTest.java

```diff
@@ -1,11 +1,59 @@
 package com.fstyle.structure_android.data.repository;
 
+import android.database.sqlite.SQLiteDatabase;
+import com.fstyle.structure_android.data.model.User;
+import com.fstyle.structure_android.data.source.local.UserLocalDataSource;
+import com.fstyle.structure_android.data.source.remote.UserRemoteDataSource;
+import com.fstyle.structure_android.data.source.remote.config.service.NameApi;
```

Comment:
ERROR: Unused import: com.fstyle.structure_android.data.source.remote.config.service.NameApi. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports